src/utils/validation/registration/referenceValidation.ts

```diff
@@ -304,8 +304,26 @@ export const reportReference = baseReference.shape({
 });
 
 // Degree
+const unconfirmedDocumentSchema = Yup.object({
+  type: Yup.string(),
+  text: Yup.string().required(resourceErrorMessage.typeRequired),
```

Comment:
Misvisende tekst
![bilde](https://github.com/user-attachments/assets/ae4c9670-aaf6-4c09-b589-9bbd8ccb7eb0)

Reply:
"Tittel på relatert resultat" e.l. er kanskje det beste? Men ingen deal-breaker for meg om det blir bare "Tittel". Kanskje Mayli har noen meninger også om du ikke har hørt der